### ISSUE TITLE
missing package inclusion in jupyter examples

### ISSUE THREAD

**Issue by sschelm** [TuringLang/Turing.jl#417]
I got an `UndefVarError: describe not defined` error when trying some of the  jupyter notebooks as they were.
The following notebooks seem to be missing the package inclusion for `describe()`, i.e. `using Mamba: describe`:
- BayesHmm.ipynb
- FHMM.ipynb
- Introduction.ipynb

**Comment by xukai92** [comment]
Thanks for pointing this out. I guess the notebooks are OK after running `using Mamba: describe`?

**Comment by sschelm** [comment]
Yes, that was implied.
Probably should have stated that explicitly, though.

Though I still get an unrelated error in the FHMM notebook, when trying to execute 
```
g = Gibbs(300, HMC(1, 0.2, 5, :m1, :T1, :m2, :T2), PG(25, 1, :s1, :s2); thin=false)
c = sample(FHMM(y), g);
```
I get the following error:
```
DomainError:
log will only return a complex result if called with a complex argument. Try log(complex(x)).
```
Hence, I can't completely check that notebook.

I also get a deprecation warnings from the GMM notebook.
```
WARNING: Array(::Type{T}, m::Int) where T is deprecated, use Array{T}(m) instead.
```

**Comment by xukai92** [comment]
The notebooks were written in previous version of Turing.jl. I'm sorry about the errors you meet.

I just created a PR to fix this bugs (https://github.com/yebai/Turing.jl/pull/419). I've test them on my local which works.

**Comment by yebai** [comment]
Closed by #419
